### ISSUE TITLE
Render control on selected mesh

### ISSUE THREAD

**Issue by manoaman** [google/neuroglancer#342]
Hi,

Does Neuroglancer support changing opacity, silhoutte levels per segmentation (mesh)?  For example, having entire segmentations of the brain transparent except one segmentation solid.

Thank you,
m

**Comment by moenigin** [comment]
I open the same volume in two different layers to achieve that...

**Comment by jbms** [comment]
Yes, what @moenigin suggested is the way to do it at the moment.

**Comment by manoaman** [comment]
Thanks @moenigin and @jbms for verifying!
